vm/systemSmartContracts/delegation.go

```diff
@@ -808,3 +1268,121 @@ func (d *delegation) IsContractEnabled() bool {
 func (d *delegation) IsInterfaceNil() bool {
 	return d == nil
 }
+
+func getTransferBackFromVMOutput(vmOutput *vmcommon.VMOutput) *big.Int {
+	transferBack := big.NewInt(0)
+	for _, outAcc := range vmOutput.OutputAccounts {
+		for _, outTransfer := range outAcc.OutputTransfers {
+			transferBack.Add(transferBack, outTransfer.Value)
+		}
+	}
+
+	return transferBack
+}
+
+func moveNodeFromList(sndList []*NodesData, dstList []*NodesData, key []byte) {
```

Comment:
this doesn't modify the actual pointer values.

Reply:
done. returned the values.